manifests/m/Microsoft/XMLNotepad/2.8.0.55/Microsoft.XMLNotepad.installer.yaml

```diff
@@ -1,8 +1,8 @@
-# Created using wingetcreate 0.4.3.1
+# Created using wingetcreate 0.4.4.1
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 
 PackageIdentifier: Microsoft.XMLNotepad
-PackageVersion: 2.8.0.52
+PackageVersion: 2.8.0.55
 Platform:
 - Windows.Universal
 - Windows.Desktop
```

Comment:
```suggestion
```
These aren't necessary since they are specified at the Installer Level. Seems to be related to microsoft/winget-create#211